docs/clientsidequeries.md

```diff
@@ -0,0 +1,85 @@
+---
+id: clientqueries
+title: Client-side Queries
+---
+
+Additionaly graphback also allows users to generate client-side queries based on the data model.
+These can be used by client side frameworks like [uniform.tools](https://uniforms.tools) to create
```

Comment:
I would move this to separate . chapter here. 
Something like

## Autogenerating forms based on server-side schema

Reply:
Then we will collaborate with @radekme to get proper integration

Reply:
Then we will collaborate with @radekmie to get proper integration

Reply:
Sure